src/governance_token.cairo

```diff
@@ -2,24 +2,27 @@ use core::traits::TryInto;
 use starknet::ContractAddress;
 
 #[starknet::interface]
-trait IToken<TStorage> {
-    // ERC20 methods
+trait IERC20<TStorage> {
     fn name(self: @TStorage) -> felt252;
     fn symbol(self: @TStorage) -> felt252;
     fn decimals(self: @TStorage) -> u8;
     fn total_supply(self: @TStorage) -> u256;
+    fn totalSupply(self: @TStorage) -> u256;
     fn balance_of(self: @TStorage, account: ContractAddress) -> u256;
+    fn balanceOf(self: @TStorage, account: ContractAddress) -> u256;
     fn allowance(self: @TStorage, owner: ContractAddress, spender: ContractAddress) -> u256;
     fn transfer(ref self: TStorage, recipient: ContractAddress, amount: u256) -> bool;
     fn transfer_from(
         ref self: TStorage, sender: ContractAddress, recipient: ContractAddress, amount: u256
     ) -> bool;
-    fn approve(ref self: TStorage, spender: ContractAddress, amount: u256) -> bool;
-    fn increase_allowance(ref self: TStorage, spender: ContractAddress, added_value: u256) -> bool;
```

Comment:
is this related to your SNIP proposal?

Reply:
no approve just got moved down

i removed increase allowance and decrease allowance because i think they're bad practice on starknet though, and i don't think they're even in the snip